studio/stores/UiStore.ts

```diff
@@ -73,21 +73,27 @@ export default class UiStore implements IUiStore {
 
   load() {
     if (typeof window === 'undefined') return
-    const savedTheme = (window.localStorage.getItem('theme') ?? 'dark') as 'dark' | 'light'
-    this.setTheme(savedTheme)
+    const localStorageThemeOption = window.localStorage.getItem('theme')
+    if (localStorageThemeOption === 'system')
+      return this.setTheme(
+        window.matchMedia('(prefers-color-scheme: dark)').matches ? 'dark' : 'light'
+      )
+    if (localStorageThemeOption === 'light') return this.setTheme('light')
+    window.localStorage.setItem('theme', 'dark')
+    this.setTheme('dark')
   }
 
   toggleTheme() {
```

Comment:
@harshcut probably good to double check this but I dont think we no longer use this `toggleTheme` method anymore now that we're moving away from the toggle in the NavigationBar. can remove this